pmd-kotlin/src/main/java/net/sourceforge/pmd/lang/kotlin/rule/xpath/internal/BaseContextNodeTestFun.java

```diff
@@ -55,10 +64,87 @@ public ExtensionFunctionCall makeCallExpression() {
             @Override
             public Sequence call(XPathContext context, Sequence[] arguments) throws XPathException {
                 Node contextNode = ((AstElementNode) context.getContextItem()).getUnderlyingNode();
-                boolean hasChildren = contextNode.getNumChildren() == Integer.parseInt(arguments[0].head().getStringValue());
-                return BooleanValue.get(klass.isInstance(contextNode) && hasChildren);
+                return BooleanValue.get(klass.isInstance(contextNode) && checker.test(arguments[0].head().getStringValue(), (T) contextNode));
             }
         };
     }
 
 }
+
+class TestUtil {
+    private TestUtil() {
+        // utility class
+    }
+
+    public static boolean hasImport(final @NonNull String reqPackage, final @NonNull Node node) {
+        // assuming root is /KotlinFile, if not, add getFirstChild()
+        boolean aHeaderHasImport = false;
+        Node root = node.getRoot();
+        int numChildren = root.getNumChildren();
+        for (int i = 0; i < numChildren; i++) {
+            if (root.getChild(i) instanceof KotlinParser.KtImportList) {
```

Comment:
Maybe use import static for readability? Can use `KtImportList` instead of `KotlinParser.KtImportList`?